### ISSUE TITLE
Add code linting

### ISSUE THREAD

**Issue by EvanHahn** [brannondorsey/host-validation#5]
It'd be great to add some code linting to enforce consistent code style. I use the somewhat-controversial [Standard](https://standardjs.com/) for Helmet, but I think some ESLint-derived code linting would be helpful.

Happy to help discuss what lint setup to use and am happy to make a pull request to add it!

**Comment by brannondorsey** [comment]
Yeah, I'm open to this! I've used jshint in the past, however I am far from an authority about best practices with linting. @EvanHahn, your input (as well as PR contributions) are welcome.

**Comment by EvanHahn** [comment]
Personally, I like Standard because it requires no configuration. Would you be okay with that?
